usecases/backup/handler.go

```diff
@@ -172,6 +172,11 @@ func (m *Manager) Restore(ctx context.Context, pr *models.Principal,
 		Status:  &status,
 		Path:    destPath,
 	}
+	// make sure there is no active restore
+	if prevID := m.restorer.lastStatus.renew(req.ID, time.Now(), destPath); prevID != "" {
+		err := fmt.Errorf("restore %s already in progress", prevID)
```

Comment:
why are we using here `fmt.Errorf` instead of `errors.Errorf`?

Reply:
Because `fmt.Errorf()` is the standard one.

errors.Errorf is part of https://github.com/pkg/errors which is no longer actively maintained.